src/daemons/rhsm_d.py

```diff
@@ -170,11 +171,9 @@ def check_status(self):
         """
         log.debug("D-Bus interface com.redhat.SubscriptionManager.EntitlementStatus.check_status called")
         ret = check_status(self.force_signal)
-        if (ret != self.last_status):
-            debug("Validity status changed, fire signal")
-            #we send the code out, but no one uses it at this time
-            self.entitlement_status_changed(ret)
-        self.last_status = ret
+        debug("Validity status changed, fire signal")
+        #we send the code out, but no one uses it at this time
+        self.entitlement_status_changed(ret)
```

Comment:
@alikins looks like this worked for @cnsnyder . At boot up time, turns out we need a signal whether the status changed or not.